src/data-storage/insert-trade.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient, Trade } from '@prisma/client';
+import { CreateTradeInput } from '../types';
+
+export const insertTrade = async (trade: CreateTradeInput) => {
+  const prisma = new PrismaClient();

```

Comment:
Functional: We should look to moving the `PrismaClient` initialization to a high level so that we can reuse it instead of constantly recreating it.